docs/documentation/actions/kafka.md

```diff
@@ -0,0 +1,189 @@
+    |          | `topics`     | List<String>        | Topics to be created in the broker |
+    |          | `properties` | Map<String, String> | Broker properties |
+
+=== "Outputs"
+
+    |       Name    | Type                                                                                                                                     |
+    |:------------- |:-----------------------------------------------------------------------------------------------------------------------------------------|
+    | `kafkaBroker` | [EmbeddedKafkaBroker](https://docs.spring.io/spring-kafka/api/org/springframework/kafka/test/EmbeddedKafkaBroker.html){:target="_blank"} |
+
+### Example
+
+=== "Kotlin"
+``` kotlin
+KafkaBrokerStartAction(
+            topics = listOf(
+                "my.queue"
+            )
+)
+```
+## Stop
+This action stop the given embedded Kafka broker. It's implicitly declared by the start action and will be executed at the end of the scenario.
+
+=== "Inputs"
+
+    | Required | Name         | Type                                                                                                                                     | Default                            |
+    |:--------:|:-------------|:-----------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------|
+    |    *     | `broker`     | [EmbeddedKafkaBroker](https://docs.spring.io/spring-kafka/api/org/springframework/kafka/test/EmbeddedKafkaBroker.html){:target="_blank"} |                                    |
+
+=== "Outputs"
+    No output
+
+# All in one basic example:
+* Create target as described [here](/documentation/actions/kafka/#:~:text=Kafka%20target%20example). 
+* Scenario example:
+
+=== "Kotlin: doesn't work" 
+``` Kotlin
+Scenario(title = "Kafka actions basic usage") {
+    Given("I start my broker") {
+        KafkaBrokerStartTask(
+            port =  "60962",
+            topics = listOf(
+                "my.queue"
+            )
+        )
+    }
+    And("I publish my favorite movie") {
+        KafkaBasicPublishTask(
+            target = "my_kafka_target",
+            topic = "my.queue",
+            headers = mapOf(
+                "contentType" to "application/json",
+                "season" to 1
+            ),
+            payload = """
+                        {
+                          "title": "Castle in the Sky",
+                          "director": "Hayao Miyazaki",
+                          "rating": 78
+                        }
+                       """.trimIndent()
+        )
+    }
+
+    When ("I request my favorite movie") {
+        KafkaBasicConsumeAction(
+            target = "my_kafka_target",
+            topic = "my.queue",
+            group = "my.group",
+            timeout = "10 sec",
+            selector = "\$..[?(\$.payload.title==\"Castle in the Sky\")]",
```

Comment:
Maybe here explain, or elaborate a little on, that the paths have to begin with `payload` or `headers`.
I don't know if we could simplify because the selector name indicates already where the path should be applied.

Reply:
In input section , I said that selector must be json path or xml path and Ileft a link to how to write paths